tapdb/assets_store.go

```diff
@@ -358,6 +369,19 @@ type ChainAsset struct {
 	// AnchorTapscriptSibling hash is equal to the Taproot root hash of the
 	// anchor output.
 	AnchorTapscriptSibling []byte
+
+	// AnchorLeaseOwner is the identity of the application that currently
```

Comment:
Not sure this needs to be part of the `ChainAsset` struct, but still getting through the diff.

Reply:
Also perhaps we can constrain this a bit further, I think it's meant to be a `[32]byte`?

Reply:
I think it's useful to eventually show this in the asset list RPC. That way we don't need an explicit `ListAssetLeases` RPC, we can just add the `leased` boolean as a filter on the `ListAssets` request params.